app/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_iaa.rb

```diff
@@ -0,0 +1,194 @@
+module Db
+  module MonthlySpAuthCount
+    module UniqueMonthlyAuthCountsByIaa
+      module_function
+
+      # @param [String] iaa
+      # @param [Symbol] aggregate (one of :sum, :unique, :new_unique)
+      # @return [PG::Result]
+      def call(iaa:, aggregate:)
```

Comment:
The individual pieces that build the query make sense, but it's not clear to me how they fit together and what is ultimately produced. Would it be possible to add documentation that describes some of the billing logic and report?

Reply:
Added some comments in 47ec4ecd1536dbb436f45947877cc6d8011fc73e... do they cover what you're looking for?